packages/sdk-vue/src/createFusionAuth/createFusionAuth.ts

```diff
@@ -19,16 +21,16 @@ export const createFusionAuth = (config: FusionAuthConfig): FusionAuth => {
   });
 
   const isLoggedIn = ref<boolean>(core.isLoggedIn);
-  const userInfo = ref<UserInfo | null>(null);
+  const userInfo = ref<T | null>(null) as Ref<UserInfo>;
```

Comment:
Can we please use `const userInfo Ref<T | null> = ref(null)`?
[This thread](https://github.com/vuejs/core/discussions/9564) has an answer from a vue contributor explaining that `ref<>()` with generic types causes issues.